doc/icingadb-migration.example.yml

```diff
@@ -0,0 +1,25 @@
+icinga2:
+  # Content of /var/lib/icinga2/icingadb.env
+  env: "da39a3e..."
+
+# IDO database
+ido:
+  type: mysql # or "pgsql" for PostgreSQL
```

Comment:
... by switching the DB types here. :)

(After all this one's migrating from IPv4 to IPv6 as well.)

Reply:
As we support both databases, I don't think we should incentivize a switch here. So if a user has the desire to switch, they will probably look for it, others may be confused why the example migrates from MySQL to PostgreSQL.

Reply:
Oh, **I think** we should incentivize a switch – from PostgreSQL **to MySQL**.

Reply:
No. It would make no sense for Icinga DB to support PostgreSQL if we would recommend to migrate away from it.